### ISSUE TITLE
Fish shell doesn't apply changes - Message 403, forbidden

### ISSUE THREAD

**Issue by celicoo** [fish-shell/fish-shell#2736]
i'm trying to customize fish shell with the fish_config, but the changes made in the localhost:8000 doesn't work, and this message appears:

```
Web config started at 'file:///Users/marceloalves/.cache/fish/web_config-X5PO3F.html'. Hit enter to stop.
::1 - - [17/Feb/2016 20:06:40] code 403, message Forbidden, path /favicon.ico

```

And i'm not finding anything related in google. I'm trying to set the solarized dark, and after several attempts this message is appearing every time i try to clear or open a new tab:

```
fish: Unknown command 'hg'
fish: hg summary >/dev/null ^/dev/null
      ^
in function '_is_hg_repo'
    called on standard input

in function '_repo_type'
    called on line 2 of file ~/.config/fish/functions/fish_prompt.fish

in command substitution
    called on line 0 of file ~/.config/fish/functions/fish_prompt.fish

in function 'fish_prompt'
    called on standard input

in command substitution
    called on standard input
```
What should i do? I'm using El Capitan.

Thanks.

**Comment by ridiculousfish** [comment]
Can you give some more details about what exactly you do, and what you expect to see versus what you actually see?

Regarding the "unknown command hg," from your prompt, it looks like that's the robbyrussell prompt. Mercurial is not installed and that makes it spew. You should be able to just switch to a different prompt.

**Comment by celicoo** [comment]
Yeap, the weird message was because robbyrussell, i just changed to other prompt and now is fine.
So, the real problem here is with the theme, i would like to use the solarized dark, but when i choose in the localhost:8000 and press "Set Theme w/o Background Color", nothing happens. Am i missing something?

Thanks @ridiculousfish

**Comment by ridiculousfish** [comment]
You might be expecting the background color of your terminal to change. This won't work because fish can't change the background color - see #2421. 

What should change are the syntax highlighting colors. Do you see those set to the correct values?

**Comment by celicoo** [comment]
Hmmmm.. makes sense now! I'm seeing the colors change. Thanks and i'm sorry for bothering u.

**Comment by bucaran** [comment]
I think Fish could improve a bit here by making it more obvious that you are just editing the syntax **color palette** with `fish_config`, to actually change the background, you would need to edit your terminal application's color settings. 

For example, selecting Tomorrow Night, will **not** make your background dark grey brown, that picture is just to give you an idea of how things would look with the selected color palette.

I know this is obvious for you veterans, but it took me a while to figure this out when I started with Fish.

**Comment by ridiculousfish** [comment]
@bucaran that's what we tried to do in #2421 :)

**Comment by krader1961** [comment]
@bucaran: Are you working with a fish binary built recently? I ask because I fixed that a few weeks ago (see #2421).

**Comment by bucaran** [comment]
Great news, thanks @krader1961
